src/app/containers/RadioSchedule/index.stories.jsx

```diff
@@ -10,12 +11,14 @@ import WithTimeMachine from '#testHelpers/withTimeMachine';
 
 // Currently, only these services have radio schedule data
 const validServices = [
+  'afrique',
```

Comment:
We've added a new service here but we haven't added fixture data for it. for storybook and local.

Reply:
Thanks Khoa. Fixture data for afrique has now been added